src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/Extensions/DataSetVersionExtensions.cs

```diff
@@ -0,0 +1,11 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Extensions;
+
+public static class DataSetVersionExtensions
+{
+    public static bool CanBeDeleted(this DataSetVersion dataSetVersion) => dataSetVersion.Status.IsDeletableState();
+
+    public static bool IsDeletableState(this DataSetVersionStatus status) => status is DataSetVersionStatus.Failed
+        or DataSetVersionStatus.Mapping
+        or DataSetVersionStatus.Draft
+        or DataSetVersionStatus.Cancelled;
+}
```

Comment:
Not sure why we've gone for extensions here when we can just have these as methods on the model itself?

Reply:
I've moved them to the model